packages/storybook/package.json

```diff
@@ -33,9 +33,9 @@
     "@fluentui/theme-samples": "8.1.5",
     "@microsoft/applicationinsights-react-js": "~3.0.5",
     "@microsoft/applicationinsights-web": "~2.6.2",
-    "@uifabric/react-hooks": "~7.13.11",
     "copy-to-clipboard": "~3.3.1",
-    "resize-observer-polyfill": "~1.5.1"
+    "resize-observer-polyfill": "~1.5.1",
+    "@fluentui/react-hooks": "~8.3.8"
```

Comment:
can we order the dependencies alphabetically so all of the @fluentui dependencies are together?